_posts/2020-10-29-cockpit-231.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Cockpit 231
+author: marusak
+date: 2020-10-29
+tags: cockpit linux
+slug: cockpit-231
+category: release
+summary: Cockpit with new performance graphs
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly. Here are the release notes from Cockpit version 231.
+
+### Replace system's graph page with a completely new USE method page
+
+System graphs (reachable after clicking on 'View graphs' on the overview page) were replaced with
+a dedicated page for the analyzing current and the historical performance of a machine.
+This page (reachable after clicking on 'View details and history' on the overview page)
+shows current metrics as well as historical events based on the
+[USE](http://www.brendangregg.com/usemethod.html) method.
+
+![performance_graphs](/images/perf-graphs.png){:.screenshot}
+
+### Machines: Reimplement the design of the main VMs list
+
+Details of virtual machines are now displayed on a separate page that contains all information in one
+place. There is no need anymore to change between tabs to see different properties of a virtual machine.
+
+![VM subpage](/images/vm-details.png){:.screenshot}
+
+### Logging of remote IP addresses
+
+Cockpit now includes the remote IP address when logging user logins (and login attempts). This means that the IP address will appear in the system logs, in the output of tools such as w or who, and in the login-time messages about when the last successful login was, and any failed login attempts. It also means that PAM modules can now filter logins based on IP address (for example, temporarily blocking a subnet when there are too many failed login attempts).
```

Comment:
We had this feature before, it was dropped with the introduction of cockpit-tls.

```suggestion
Cockpit records the remote IP address for user logins (and failed attempts) again. The IP address will appear in the system logs, in the output of tools such as `w` or `who`, and in the login-time messages about when the last successful login happened, and any failed login attempts. It also means that PAM modules can filter logins based on IP address (for example, temporarily blocking a subnet when there are too many failed login attempts).
```